### ISSUE TITLE
Position not being added to Session.set('location);

### ISSUE THREAD

**Issue by kensayers75** [thespacedojo/cityforks-2#1]
Hi,

Hope you can help...

I first got this problem following your o'riley screencast.  Session.set won't add the returned position. your screencast (at 34:10) shows you going into the console and checking out Session.get('location'). I just got an empty object here. I get this same if I download cityforks 1 and 2. If I console.log position i can see the object and coords etc but it just won't add to the Session. 

---

This application contains Mongol. Read more: https://github.com/msavin/Mongol
meteor.js:888 Exception from Tracker recompute function:
meteor.js:888 TypeError: Cannot read property 'latitude' of undefined
    at null.<anonymous> (home.js:27)
    at blaze.js:1865
    at Function.Template._withTemplateInstanceFunc (blaze.js:3476)
    at blaze.js:1864
    at Object.Blaze._withCurrentView (blaze.js:2197)
    at viewAutorun (blaze.js:1863)
    at Tracker.Computation._compute (tracker.js:327)
    at Tracker.Computation._recompute (tracker.js:346)
    at Object.Tracker._runFlush (tracker.js:485)
    at onGlobalMessage (meteor.js:372)

Thanks
Ken

**Comment by queso** [comment]
Yeah, I am not sure why, but it looks like the position object has changed some... My latest iteration fixes this issue: https://github.com/meteorclub/cityforks-4
